### DISCUSSION TITLE
Alphabetical sort in production Storybook is sorting Alphabetically

### DISCUSSION THREAD

**Discussion by catdc2017** [storybookjs/storybook#17886] [Help] [answered]
Hi I am using storybook 13 with webpack5.71

<img width="486" alt="Screenshot 2022-04-05 at 11 53 03" src="https://user-images.githubusercontent.com/26303634/161738863-1546e146-16d4-4cf1-852f-4a9c29383b6d.png">

I am having an issue with the sort order in production.  Locally is sorts correctly but in production it is looking like this below

<img width="176" alt="Screenshot 2022-04-05 at 10 57 33" src="https://user-images.githubusercontent.com/26303634/161738951-de547318-6bbc-4d1d-b700-c39c2e7a511c.png">

I included numbers and special characters to try to force the sort order but it is sorting alphabetically and not respecting numbers or special characters?

**Comment by KatWorkGit** [comment]
Do you mean that you are using _Angular_ 13? Storybook is only on version 7.5. What version of storybook are you using?

I was looking for help with a storySort issues and saw this comment about angular, but I'm not sure if it is related at all 😕 
https://github.com/storybookjs/storybook/discussions/22529#discussioncomment-7521217

**Comment by chantastic** [comment, marked as answer]
this doc is probably what you're looking for: https://storybook.js.org/docs/7.5/configure/sidebar-and-urls

special characters and numbers are unpredictable.
they should definitely be consistent between local and production.
but, indifferent to that, the doc above has details on configuring order without the addition of special characters and ordinal numbers.
